src/main/java/me/arminb/sara/controller/models/QuestionRequest.java

```diff
@@ -103,9 +70,9 @@ public Question toQuestion() {
         Question question = new Question();
         question.setTitle(title);
         question.setRate(rate);
-        question.setDate(new Date());
         question.setContent(content);
-        question.setAnswers(answers);
+        question.setModifiedAt(new Date());
+        //question.setCreatedAt(?);
```

Comment:
During creating or updating an entity you just need to set modified at. Created at will not be saved as a separate field in the database. When a document is loaded from the database, its corresponding createdAt can be calculated from its object id